homeassistant/components/fibaro/climate.py

```diff
@@ -159,7 +161,11 @@ def __init__(self, fibaro_device):
                 self._attr_supported_features |= ClimateEntityFeature.TARGET_TEMPERATURE
                 tempunit = device.properties.unit
 
-            if "setMode" in device.actions or "setOperatingMode" in device.actions:
+            if (
```

Comment:
This could be slightly simplified:

```python
if any(action for action in ("setMode", "setOperatingMode", "setThermostatMode") if action in device.actions)
```

Reply:
For me as Java developer this looks strange but for python developers it looks most probably better. I changed it accordingly.